views/room.ejs

```diff
@@ -13,17 +13,12 @@
     <!--[if IE]><link rel="shortcut icon" href="/favicon.ico"><![endif]-->
     <link rel="icon" href="/favicon.png">
 
-    <link rel="stylesheet prefetch" href="/css/endMeeting.opentok.css">
-
     <link rel="stylesheet prefetch" href="/css/room.opentok.css">
-    <link rel="stylesheet prefetch" href="/css/buttons.css" />
-    <link rel="stylesheet prefetch" href="/css/landing.opentok.css">
```

Comment:
Unfortunately, removing landing.opentok.css breaks the room view. To test, go to a room URL (such as http://localhost:8123/room/foo) directly, without coming from the root landing page. The UI is broken (e.g., the preview video is squashed). I'll add a ticket to fix this (so that the styles in landing.opentok.css only apply to the landing page, and the precall styles are in a separate stylesheet that can be loaded by landing and room views.

Reply:
this is being handled by https://github.com/opentok/opentok-rtc/pull/667.

Merge conflict should resolve this